proposals/nnnn-android-studio-sdl-plugin.md

```diff
@@ -0,0 +1,51 @@
+#Android Studio IDE SDL Plugins
+
+*	Proposal : SDL -nnnn 
+*	Author : Michael Morgan
+*	Status:  Awaiting Review
+*	Review Manager : TDB
+*	Impacted Platforms : Android
```

Comment:
Please copy / paste the [template](https://github.com/smartdevicelink/sdl_evolution/blob/master/0000-template.md) header here exactly (especially the proposal name, author, and status sections are done incorrectly).